src/routes/safe/components/Transactions/TxList/QueueTransactions.tsx

```diff
@@ -32,16 +33,19 @@ export const QueueTransactions = (): ReactElement => {
   }
 
   return (
-    <TxsInfiniteScroll next={next} hasMore={hasMore} isLoading={isLoading}>
-      {/* Next list */}
-      <TxLocationContext.Provider value={{ txLocation: 'queued.next' }}>
-        {transactions.next.count !== 0 && <QueueTxList transactions={transactions.next.transactions} />}
-      </TxLocationContext.Provider>
+    <>
+      <BatchExecute />
```

Comment:
I'm not sure about this placement. Have you spoken with @liliiaorlenko? I think it should at least be moved down a bit.